roles/keystone/templates/etc/keystone/policy.json

```diff
@@ -2,9 +2,11 @@
 
 {
 {% if policy_for_horizon|default('False')|bool %}
-    "admin_required": "role:admin or is_admin:1 or role:cloud_admin or role:project_admin or rule:owner",
+    "admin_required_limit": "role:admin or is_admin:1 or role:cloud_admin or role:project_admin or rule:owner",
```

Comment:
for policy_for_horizon is false, "admin_required_limit" = "admin_required", and cloud_admin has CRUD, same as before.
for policy_for_horizon is true, "admin_required_limit" can do CRUD, but "admin_required" not.
cloud_admin has no CRUD